test/powershell/Language/Scripting/ForeachParallel.Tests.ps1

```diff
@@ -31,11 +24,21 @@ Describe "Parallel foreach syntax" -Tags "CI" {
         It '$errors[0].ErrorId' { $errors[0].ErrorId | Should -BeExactly 'InvalidForeachFlag' }
     }
 
-    Context 'Generate an error on -parallel that is not a workflow' {
+    Context 'Generate an error on -parallel' {
         $errors = @()
         $ast = [System.Management.Automation.Language.Parser]::ParseInput(
             'foreach -parallel ($input in $bar) { }', [ref]$null, [ref]$errors)
         It '$errors.Count' { $errors.Count | Should -Be 1 }
-        It '$errors[0].ErrorId' { $errors[0].ErrorId | Should -BeExactly 'ParallelNotSupported' }
+        It '$errors[0].ErrorId' { $errors[0].ErrorId | Should -Be 'KeywordParameterReservedForFutureUse' }
     }
-}
\ No newline at end of file
+
+    Context 'Generate an error on -throttlelimit' {
+        $errors = @()
+        $ast = [System.Management.Automation.Language.Parser]::ParseInput(
+            'foreach -throttlelimit 2 ($input in $bar) { }', [ref]$null, [ref]$errors)
+        It '$errors.Count' { $errors.Count | Should -Be 2 }
+        It '$errors[0].ErrorId' { $errors[0].ErrorId | Should -Be 'KeywordParameterReservedForFutureUse' }
+        It '$errors[1].ErrorId' { $errors[1].ErrorId | Should -Be 'ThrottleLimitRequiresParallelFlag' }
```

Comment:
As I note above the error makes no sense.

Reply:
See comment above.